src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ModelExplorerExtensions.cs

```diff
@@ -40,6 +41,22 @@ public static string GetSimpleDisplayText(this ModelExplorer modelExplorer)
                 return modelExplorer.Metadata.NullDisplayText;
             }
 
+            if (modelExplorer.Metadata.IsEnum && modelExplorer.Model is Enum modelEnum)
+            {
+                var enumStringValue = modelEnum.ToString("d");
```

Comment:
Should this specify `InvariantCulture`?

Reply:
The `Enum.ToString(...)` overloads taking an `IFormatProvider` are both `[Obsolete]`.